packages/site/src/layout/AnimatedPresenceDisclosure.tsx

```diff
@@ -14,56 +15,61 @@ function AnimatedPresenceDisclosure({
   to,
 }: AnimatedPresenceDisclosureProps) {
   const [isOpen, setIsOpen] = useState(false);
+  const [isHovered, setIsHovered] = useState(false);
 
   const handleButtonClick = (e: React.MouseEvent) => {
     e.preventDefault();
     setIsOpen(!isOpen);
   };
 
+  const handleHover = (e: React.MouseEvent) => {
+    e.preventDefault();
+    setIsHovered(!isHovered);
+  };
+
   return (
-    <>
-      <Link
-        to={to ?? "/"}
-        href="https://www.google.com"
-        tabIndex={0}
-        onFocus={e => (e.currentTarget.style.boxShadow = "0 0 3px #0073e6")}
-        onBlur={e => (e.currentTarget.style.boxShadow = "none")}
-        style={{
-          display: "flex",
-          alignItems: "center",
-          justifyContent: "space-between",
-          backgroundColor: "var(--color-surface--background)",
-          outline: "none",
-          textDecoration: "none",
-        }}
-      >
-        <Typography fontWeight="semiBold" size="large" textColor="heading">
-          {title}
-        </Typography>
-        <button
-          type="button"
-          onClick={handleButtonClick}
+    <div>
+      <span className={styles.disclosureNavItem}>
+        <Link
+          to={to ?? "/"}
+          href="https://www.google.com"
```

Comment:
It appears that `href` is not a [valid prop for Link](https://api.reactrouter.com/v7/interfaces/react_router.LinkProps.html)

Reply:
oh nice, since I'm assuming we actually don't want to send people to Google dot com can we just remove that line?

Reply:
[looks like it ](https://github.com/GetJobber/atlantis/pull/2237/commits/4c63a98006e68afd764df6d0a16fe8d8691c7d4a)